rust/pact_matching_ffi/src/log/ffi.rs

```diff
@@ -49,13 +49,14 @@ pub extern "C" fn logger_init() {
 /// Attach an additional sink to the thread-local logger.
 ///
 /// This logger does nothing until `logger_apply` has been called.
+#[allow(clippy::missing_safety_doc)]
```

Comment:
Because this documentation will really be used from the C side, I opted to put this override here rather than clutter the generated C docs with Rust information that isn't relevant in that context.

Really, this whole crate should never be used from Rust, so I could have opted to ignore the initial lint about the function not being marked unsafe as well. I suppose this raises a question of whether we want to mark _all_ the FFI functions unsafe to be consistent.